projects/swc-starter-webpack/src/example_usages/sp-switch.html

```diff
@@ -15,6 +15,18 @@ <h3>sp-switch</h3>
 <div id="content">
     <div id="left">
         <div id="subtitle">sp-switch</div>
+        <div style="display: flex; justify-content: space-between">
+            <div style="display: flex; flex-direction: column">
+                <h4 class="spectrum-Heading--subtitle1">Small Size</h4>
+                <sp-switch size="s">Small</sp-switch>
```

Comment:
Are we not adding the remaining sizes like m, l in the starter plugin?

Reply:
We can add them too, will probably add those cases.